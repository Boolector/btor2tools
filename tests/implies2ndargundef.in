1 zero 32
3 implies 32 1 2
