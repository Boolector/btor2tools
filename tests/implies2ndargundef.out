*** catbtor: parse error in 'implies2ndargundef.in' line 2: undefined argument id
